extension/chrome/elements/compose-modules/compose-recipients-module.ts

```diff
@@ -545,6 +550,7 @@ export class ComposeRecipientsModule extends ViewModule<ComposeView> {
   private searchContacts = async (input: JQuery<HTMLElement>): Promise<void> => {
     try {
       this.view.errModule.debug(`searchContacts`);
+      this.view.errModule.debug(`googleContactsSearchEnabled: ${await this.googleContactsSearchEnabled}`);
```

Comment:
why would we pause searchContacts here?
we can exit early at line 561/567 etc.?

Reply:
This condition was added because I need some sort of debugging.
Can remove this line

Reply:
Please remove